skbio/sequence/_grammared_sequence.py

```diff
@@ -7,9 +7,10 @@
 # ----------------------------------------------------------------------------
 
 from __future__ import absolute_import, division, print_function
-from future.utils import with_metaclass
 
 from abc import ABCMeta, abstractproperty
+from six import add_metaclass
```

Comment:
put this 3rd party import with numpy imports, group itertools and abc imports together b/c they're stdlib